cylc/rose/platform_utils.py

```diff
@@ -17,20 +17,22 @@
 # -----------------------------------------------------------------------------
 """Interfaces for Cylc Platforms for use by rose apps.
 """
+import subprocess
 from optparse import Values
 from pathlib import Path
-from typing import Dict, Any
+from typing import Any, Dict
 
 from cylc.flow.config import WorkflowConfig
-from cylc.flow.exceptions import PlatformLookupError
 from cylc.flow.id_cli import parse_id
+from cylc.flow.platforms import (
+    HOST_REC_COMMAND,
```

Comment:
This allows backticks, was that desired?

Reply:
This is only for the old `[remote]host` syntax which I think we supported at Cylc 7.